app/_src/policies/meshhealthcheck.md

```diff
@@ -40,7 +40,7 @@ The `MeshHealthCheck` policy supports both L4/TCP and L7/HTTP/gRPC checks.
 
 ### Protocol selection
 
-The health check protocol is selected by picking the most [specific protocol](/docs/{{ page.version }}/policies/protocol-support-in-kuma/#protocol-support-in-kuma) 
```

Comment:
why are we removing the fragment? maybe we could just not suffix it with "Kuma"?

Reply:
This fragment doesn't exist (it's just the page)